code/modules/crafting/recipes.dm

```diff
@@ -606,3 +606,13 @@
 	name = "Mummification Bandages (Body)"
 	result = /obj/item/clothing/under/mummy
 	reqs = list(/obj/item/stack/sheet/cloth = 5)
+
+/datum/crafting_recipe/guillotine
+	name = "Guillotine"
+	result = /obj/structure/guillotine
+	time = 150 // Building a functioning guillotine takes time
+	reqs = list(/obj/item/stack/sheet/plasteel = 3,
+		        /obj/item/stack/sheet/mineral/wood = 20,
+		        /obj/item/stack/cable_coil = 10)
```

Comment:
This seems like an awfully small amount of resources to make the thing overall. 
20 wood planks? It takes 20 wood planks to make a wooden buckler, and this is supposed to be a giant structure. 10 cable coils is also less than it takes to make cable restraints. 3 plasteel sheets also seems like it might be too little aswell. 
Making this thing as is easy as it is to make will take away some of the thrill and rarity of seeing it only on occasion. 
I'd recommend some of these requirements being increased a little bit.

Reply:
@lordpidey @Bawhoppen 

I can increase the amounts no problem, although I went with plasteel over metal because it's a little more rare (just partly deconstruct a reinf wall)

How does 40 wood, 25 cable and 7 plasteel sound?

Reply:
Sounds reasonable amounts to me.

Reply:
The amount of wood needed for construction is wildly inconsistent and has no real relation to the actual ingame size, so you can probably go lower than that.
For reference
- Door: 10x
- Drying rack: 10x
- Apiary: 40x
- Dresser: 10x
- Ore box: 4x
- Barricade: 5x

Reply:
i don't know you can build an electric chair which has instant death instead of delayed death with less materials that's needed here i don't tnhink a change is needed imo.